js/src/lib/Details/Markdown.js

```diff
@@ -12,24 +17,31 @@ const renderAndEscapeMarkdown = ({ source, githubRepo, gitHead }) => {
 
   if (githubRepo) {
     const { user, project, path } = githubRepo;
-    renderer.image = function(href, title, text) {
-      return `<img src="${prefixURL(href, {
-        base: 'https://raw.githubusercontent.com',
+    const prefix = (href, base) =>
+      prefixURL(href, {
+        base,
         user,
         project,
         head: gitHead ? gitHead : 'master',
         path,
-      })}" title="${title}" alt="${text}"/>`;
-    };
+      });
 
-    renderer.link = function(href, title, text) {
-      return `<a href="${prefixURL(href, {
-        base: 'https://github.com',
-        user,
-        project,
-        head: gitHead ? `tree/${gitHead}` : 'tree/master',
-        path,
-      })}" title="${title}">${text}</a>`;
+    renderer.image = (href, title, text) =>
+      `<img src="${prefix(href, GITHUB.raw)}" title="${title}" alt="${text}"/>`;
+
+    renderer.link = (href, title, text) =>
+      `<a href="${prefix(href, GITHUB.main)}" title="${title}">${text}</a>`;
+
+    renderer.html = function(html) {
+      return html
```

Comment:
It'd be a lot cleaner to use a Markdown parser that returns an AST, so that we could modify this directly in the AST rather than using string replacement. Something like [Markdig](https://github.com/lunet-io/markdig) but for JavaScript. Does an advanced Markdown parser like that exist in JavaScript?